app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  protect_from_forgery unless: -> { request.format.json? }
```

Comment:
🤔

Reply:
If you have time to discuss this, I would appreciate it